### ISSUE TITLE
Is the second return statement incorrect?

### ISSUE THREAD

**Issue by lvyilin** [ChaofWang/L-GM_loss_pytorch#3]
The implementation of paper author is: https://github.com/WeitaoVan/L-GM-loss/blob/598582f0631bac876b3eeb8d6c4cd1d780269e03/tensorflow/resnet_model.py#L472
while in your code the first return param is `neg_sqr_dist`. The variable `logits_with_margin` is not used.

**Comment by ChaofWang** [comment]
@lvyilin thank you very much. you are right. This is a mistake, I have fixed it.
